packages/client/src/pages/app/AppDetailPage.tsx

```diff
@@ -239,105 +239,99 @@ export const AppDetailPage = () => {
+                        setValue('detailsForm.markdown', parsedMeta.markdown);
+                        setValues((prev) => ({
+                            ...prev,
+                            markdown: parsedMeta.markdown || '',
+                        }));
+                        setValues((prev) => ({ ...prev, ...parsedMeta }));
+                    }
+                } else if (idx === 1) {
+                    if (res.value.type === 'error') {
+                        emitMessage(true, res.value.output);
+                    } else {
+                        if (res.value.output !== null) {
+                            setValue('markdown', res.value.output);
+                            setValue('detailsForm.markdown', res.value.output);
                             setValues((prev) => ({
                                 ...prev,
-                                markdown: parsedMeta.markdown || '',
+                                markdown: res.value.output || '',
                             }));
-                            setValues((prev) => ({ ...prev, ...parsedMeta }));
-                        }
-                    } else if (idx === 1) {
-                        if (res.value.type === 'error') {
-                            emitMessage(true, res.value.output);
-                        } else {
-                            if (res.value.output !== null) {
-                                setValue('markdown', res.value.output);
-                                setValue(
-                                    'detailsForm.markdown',
-                                    res.value.output,
-                                );
-                                setValues((prev) => ({
-                                    ...prev,
-                                    markdown: res.value.output || '',
-                                }));
-                            }
-                        }
-                    } else if (idx === 2) {
-                        if (res.value.type === 'error') {
-                            emitMessage(true, res.value.output);
-                        } else {
-                            const modelled = modelDependencies(
-                                res.value.output,
-                            );
-                            setValue('dependencies', modelled);
-                            setValue('selectedDependencies', modelled);
                         }
                     }
+                } else if (idx === 2) {
+                    if (res.value.type === 'error') {
+                        emitMessage(true, res.value.output);
+                    } else {
+                        const modelled = modelDependencies(res.value.output);
+                        setValue('dependencies', modelled);
+                        setValue('selectedDependencies', modelled);
+                    }
                 }
-            }),
-        );
+            }
+        });
```

Comment:
Could we add error checking to this as well? Does not look like we do anything if the pixel call errors out.